staging/src/k8s.io/kubectl/pkg/cmd/config/set.go

```diff
@@ -141,6 +202,311 @@ func (o setOptions) validate() error {
 	return nil
 }
 
+// modifyConfigJson: modifyConfig, but for JSON path
```

Comment:
Could you write more about what this function does? `modifyConfig` does not have documentation either so this comment is not very helpful as documentation.